### ISSUE TITLE
Add magazine well retextures

### ISSUE THREAD

**Issue by jonpas** [Theseus-Aegis/TheseusServices#118]
Magazines on this screenshot should be same as `MX_BlackCamo` retexture:
![20190907140751_1](https://user-images.githubusercontent.com/7935003/64474729-ecc31780-d178-11e9-862b-31085aec9558.jpg)

Magazine wells split that so we have to apply it to the magazines as well (as in add new classes).

**Comment by jonpas** [comment]
Scribble:
```cpp
class CfgMagazines {
    class MyBlackMagazineClass: 30Rnd_65x39_caseless_mag {
        hiddenSelectionsTextures[] = {"\x\tacs\myBlackMXTexture_co.paa"};
    };
};

class CfgMagazineWells {
    class CBA_65x39_MX {
        ADDON[] = {
            QCLASS(MyBlackMagazineClass)
        };
    };
};
```

**Comment by jonpas** [comment]
Partly fixed in #119, keeping this open as ideally we would restore the Black Camo magazines.

**Comment by jonpas** [comment]
Unit magazines fixed in #121, however proper Black Camo magazines are still not there.
